src/test/java/io/appium/java_client/android/AndroidFunctionTest.java

```diff
@@ -27,16 +27,13 @@
 
 public class AndroidFunctionTest extends BaseAndroidTest {
```

Comment:
this test does not make sense if it uses standard functions. It must be removed together with deprecated stuff